content/docs/quickstart/dart.md

```diff
@@ -26,17 +26,17 @@ binaries for your operating system (`protoc-<version>-<os>.zip`) from here:
 [https://github.com/google/protobuf/releases](https://github.com/google/protobuf/releases)
 
   * Unzip this file.
-  * Update the environment variable `PATH` to include the path to the protoc
+  * Update the environment variable `PATH` to include the path to the `protoc`
     binary file.
 
-Next, install the protoc plugin for Dart
+Next, install the `protoc` plugin for Dart
 
 ```sh
 $ pub global activate protoc_plugin
 ```
 
 The compiler plugin, `protoc-gen-dart`, is installed in `$HOME/.pub-cache/bin`.
-It must be in your $PATH for the protocol compiler, protoc, to find it.
+It must be in your `PATH` for the protocol compiler, protoc, to find it.
```

Comment:
Should this be 
```
`$PATH`
```
to make clear that this is a bash variable?

Reply:
In our Dart & Flutter docs, AFAIK, we use the environment variable _name_ only (as opposed to the expression `$foo`). For example, this is excerpt from [Running an app with the standalone Dart VM](https://dart.dev/tutorials/server/cmdline#running-an-app-with-the-standalone-dart-vm):

> Important: The location of the SDK installation directory ... **By putting this directory in your PATH** you can refer to the dart command and other commands, such as dartanalyzer, by name.

@kwalrath can correct me if I'm wrong.

Reply:
Good to know!